src/Home/PaletteEditor/PaletteEdit.ZC

```diff
@@ -152,27 +145,10 @@ U0 DrawCtrlSlider(CDC *dc, CCtrl *c)
 
 	for (i = 0; i < COLORS_NUM; i++)
 	{
-		for (j = 0; j <= size; j++)
-		{
-			for (k = 0; k <= size; k++)
-			{
-				tempColor = gr_palette[i];
-				// black borders
-				if (k < border || k > size - border || j < border || j > size - border - 1)
-				{
-					if (selected_color == i)
-						tempColor = 0xFFFFFF;
-					else
-						tempColor = 0x000000;
-				}
-				// color preview
-				DrawPixel(c->win_task->pix_left + j + (size * i), c->win_task->pix_bottom - size * 1.5 + k, tempColor);
-			}
-		}
 		// color label and text preview
 		// TODO: THIS LOGIC DOESNT WORK ON LIGHT THEMES		
 		// if color too light or too dark add bg
```

Comment:
I would personally use proper punctuation for those comments. Even if they maybe are temporary.

Reply:
unchanged, wont fix as there's plenty that needs rework in paletteedit